app/src/main/java/com/worldtreeinc/leaves/utility/EventLoaderTask.java

```diff
@@ -125,7 +125,6 @@ public void runningSearch(final String query, final String category) {
             }
         }
         eventsListAdapter = new EventsListAdapter(activity,matchedEvents, isPlanner);
-        //listView = (ListView) activity.findViewById(R.id.listView);
```

Comment:
WAY TO GO: Nice catch and good call on removing the logger library imported and the toast, as well as this commented out piece of code.